extension/llm/export/export_passes.py

```diff
@@ -0,0 +1,97 @@
+import torch
+
+from executorch.exir.pass_base import ExportPass
+from torch._subclasses import FakeTensor
+from torch.fx.passes.infra.pass_base import PassResult
+
+
+def _normalize_dims(tensor: FakeTensor, dim_0: int, dim_1: int):
+    """
+    Normalize the dimensions of a tensor.
+    """
+    assert tensor is not None, "Tensor is None"
+    ndim = tensor.ndim
+    if dim_0 < 0:
+        dim_0 = ndim + dim_0
+    if dim_1 < 0:
+        dim_1 = ndim + dim_1
+    assert dim_0 < ndim and dim_1 < ndim, f"Invalid dimensions: {dim_0}, {dim_1}"
+    return dim_0, dim_1
+
+
+class RemoveRedundantTransposes(ExportPass):
```

Comment:
Would name something else, since this is also undoing permutes as well

Reply:
Yeah good point. I should rename it to permute as transpose is special case of permute as well